pkg/downloader/manager.go

```diff
@@ -578,7 +578,8 @@ func (m *Manager) resolveRepoNames(deps []*chart.Dependency) (map[string]string,
 	missing := []string{}
 	for _, dd := range deps {
 		// Don't map the repository, we don't need to download chart from charts directory
-		if dd.Repository == "" {
+		// When OCI is used there is no Helm repository
+		if dd.Repository == "" || registry.IsOCI(dd.Repository) {
```

Comment:
You might want to rebase off `main` or otherwise undo this change. Adding this back causes an error.

Reply:
Yes thank you, will do

Reply:
cherry-picked ee73a0263c02a6c53261fdaf2ad97542b18a844c from https://github.com/helm/helm/pull/10535 back into this branch 👍